pkg/lib/bundle/validate.go

```diff
@@ -0,0 +1,147 @@
+package bundle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+// ImageValidator is a struct implementation of the Indexer interface
+type ImageValidator struct {
+	ImageReader containertools.ImageReader
+	Logger      *log.Entry
```

Comment:
Since there's a constructor for this struct, do users need access to the struct fields? Can they be unexported to make the API surface smaller?